variables.tf

```diff
@@ -15,18 +15,18 @@ variable "private_subnet_ids" {
 }
 
 variable "cluster_id" {
-  description = "AWS ECS Cluster id"
+  description = "The Amazon Resource Name (ARN) that identifies the cluster."
 }
 
-variable "task_definition_image" {
-  description = "Image for the task definition (repo:tag or repo@digest)."
+variable "task_container_image" {
```

Comment:
I think we want to keep this the same with the ecs/service module:
https://github.com/telia-oss/terraform-aws-ecs/blob/master/modules/service/variables.tf#L35

So if we change it here, then I think it needs to change in the ECS module as well.

Reply:
Yepp and I think we should, the image is not actually defined in the task definition but in the container definition.